### PR TITLE
language tagged string equality

### LINKED ISSUE #173: Support language tagged strings in claimgraph representation.
https://www.w3.org/TR/rdf11-concepts/#section-Graph-Literal
https://github.com/docknetwork/sdk/blob/1009c20b5adc96010f7029123b746c53c59ba0e7/src/utils/claimgraph.js

### PR DESCRIPTION
**Pull request by bddap**
fixes #173

### CODE REVIEW

File: src/utils/claimgraph.js [hunk 1]
```diff
@@ -23,14 +35,23 @@ export function fromJsonldjsCg(jcg) {
 
 // convert a single node from json-ld claimgraph representation to this module's representation.
 function fromJsonldjsNode(jn) {
+  let ret;
```
Comment (SamHellawell) [inline src/utils/claimgraph.js:38]: `ret` declaration can go below as its not used until `case 'Literal'`

File: src/utils/claimgraph.js [hunk 2]
```diff
@@ -23,14 +35,23 @@ export function fromJsonldjsCg(jcg) {
...
   switch (jn.termType) {
     case 'NamedNode':
       return { Iri: jn.value };
     case 'BlankNode':
       return { Blank: jn.value };
     case 'Literal':
       assert(jn.datatype.termType === 'NamedNode', 'The datatype of an RDF literal must be an IRI');
-      return { Literal: { value: jn.value, datatype: jn.datatype.value } };
+      ret = { Literal: { value: jn.value, datatype: jn.datatype.value } };
```
Comment (SamHellawell) [inline src/utils/claimgraph.js:46]: you can use `const ret` here, as the reference of `ret` never changes (it remains that object, you are just mutating its values)

**Review by SamHellawell — APPROVED** [review]
lgtm just minor feedback

File: src/utils/claimgraph.js (hunk 1 above)
Comment (bddap) [reply src/utils/claimgraph.js:38]: I did that originally but eslint got upset. It doesn't like block-scoped variables to be declared inside switch statements.

**Pull request merged**